src/components/StatsPanel/StatsPanel.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { StatsPanel } from './StatsPanel';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof StatsPanel> = {
+  title: 'Dashboard/StatsPanel',
+  component: StatsPanel,
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+export default meta;
```

Comment:
This file uses `export default meta;` (Storybook's required default export). Project rules in the repo specify 'Use named exports (NO default exports)'. This is a conflict between the project's rule and Storybook CSF requirements. The rule is strict and this line violates it.

**Sugerencia:**
```typescript
If the project rule is strict, add an exception for Storybook meta default exports in linting or documentation. If no exception will be added, consider documenting the exception in a README for stories. Otherwise, keep the default export because Storybook requires it, but update the project lint/rules to permit this pattern.
```